en/_posts/2016-02-25-apache-logs.md

```diff
@@ -6,6 +6,14 @@ author: Babacar Diassé
 lang: en
 ---
 
+<div class="notice--warning" style="border: 1px #CE6D00 solid;padding: 10px;">
+  <h4 style="margin-top:0;">DISCLAIMER</h4>
+  <p>Following procedure have the advantage to be setup very quickly but is NOT recommended at all in production as it can block Apache process in case of heavy traffic.</p>
+  <p>
+        In production, please use non blocking solution as this one: 
+        [Shipping logs to PaaS Logs with Filebeat](/kb/en/logs/filebeat-logs.html) 
```

Comment:
Does the link in the <p> render well on jekyll ? It does not on https://github.com/pdepaepe/knowledge-base/blob/logs/en/_posts/2016-02-25-apache-logs.md